admin/howto/manage-users.md

```diff
@@ -0,0 +1,45 @@
+# Manage access to the hub
+
+The **Administrator Panel** can be used to maintain the list of users
```

Comment:
We should link to this section from the "User authentication and authorization" section

Reply:
I can't seem to figure out how to link from one page to another :| Plain links don't seem to work (with or without .md), and neither do target crosslinks. Help?